src/ui/settings/components/InstallPluginButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { stylesheet } from "@metro/common";
+import { General } from "@ui/components";
+import { getAssetIDByName } from "@ui/assets";
+import { showInstallPluginAlert } from "./InstallPluginAlert";
```

Comment:
This import should use our import aliases, e.g. `@ui/settings/components/InstallPluginAlert`.